sentry_sdk/client.py

```diff
@@ -128,7 +128,10 @@ def _send_sessions(sessions):
             self.integrations = setup_integrations(
                 self.options["integrations"],
                 with_defaults=self.options["default_integrations"],
-                with_auto_enabling_integrations=self.options["_experiments"].get(
+                with_auto_enabling_integrations=self.options[
+                    "auto_enabling_integrations"
+                ]
+                or self.options["_experiments"].get(
                     "auto_enabling_integrations", False
                 ),
```

Comment:
let's get rid of this flag, there is no backwards compat concern if auto enabling integrations are becoming the default